### PR TITLE
Extract `ReservedPrefix` constant

### PR DESCRIPTION
**Pull request by AlekSi**
# Description

Just a small clean-up before we use it in more places.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [ ] I added/updated integration/compatibility tests (and they pass).
- [x] I added/updated comments and checked rendering.
- [x] I made spot refactorings.
- [ ] I updated user documentation.
- [x] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Milestone (`Next`), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: internal/backends/sqlite/metadata/metadata.go [hunk 1]
```diff
@@ -32,7 +33,7 @@ import (
 
 const (
 	// DefaultColumn is a column name for all fields.
-	DefaultColumn = "_ferretdb_sjson"
+	DefaultColumn = backends.ReservedPrefix + "_sjson"
```
Comment (rumyantseva) [inline internal/backends/sqlite/metadata/metadata.go:36]: Some tests start failing due to this test. `DefaultColumn` used to be `_ferretdb_sjson`, but now it's `_ferretdb__sjson`. Should it remain `_ferretdb_sjson` or do we use double `_` on purpose (if so, tests need to be updated)?

File: internal/backends/sqlite/metadata/registry.go [hunk 2]
```diff
@@ -42,7 +42,7 @@ const (
 	reservedTablePrefix = "sqlite_"
 
 	// SQLite table name where FerretDB metadata is stored.
-	metadataTableName = "_ferretdb_collections"
+	metadataTableName = backends.ReservedPrefix + "_collections"
```
Comment (rumyantseva) [inline internal/backends/sqlite/metadata/registry.go:45]: Same here: now `metadataTableName` has double `_` in the name.

File: internal/backends/sqlite/metadata/metadata.go (hunk 1 above)
Comment (rumyantseva) [reply internal/backends/sqlite/metadata/metadata.go:36]: P.S. I guess it's not on purpose as this change breaks back compatibility. Fun fact: integration tests would pass. That's why I avoid using constants in unit tests: sometimes constant-less tests help catch such things!

**Review by chilagrow — APPROVED** [review]

**Review by rumyantseva — APPROVED** [review]
:shipit:

**Pull request merged**